src/NuGet.Clients/PackageManagement.VisualStudio/Utility/LegacyCSProjProjectReference.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Frameworks;
+using System;
+
+namespace NuGet.PackageManagement.VisualStudio
+{
+    /// <summary>
+    /// A data transfer type for calls to <see cref="EnvDTEProjectAdapter" />.
+    /// </summary>
+    public class LegacyCSProjProjectReference
+    {
+        public string UniqueName;
+        public Array MetadataElements;
```

Comment:
Ideally a strong type here

---
In reply to: [84561395](https://github.com/NuGet/NuGet.Client/pull/950#discussion_r84561395) [](ancestors = 84561395)